src/coreclr/vm/fcall.h

```diff
@@ -18,47 +18,8 @@
 // turned on again when the frame is torn down.   This mechanism should
 // be sufficient to ensure that the rules are followed.
 
```

Comment:
This may also need to be reworded. If I'm understanding correctly, it will be impossible to do anything requires a frame inside FCall. The comment here can be "the mechanism to setup a frame has been removed" and "use QCall instead".